module/attribute/src/Application/Validator/AttributeCodeConstraint.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Attribute\Application\Validator;
+
+use Ergonode\Attribute\Domain\ValueObject\AttributeCode;
+use Ergonode\SharedKernel\Application\Validator\SystemCodeConstraint;
+
+/**
+ * @Annotation
+ */
+class AttributeCodeConstraint extends SystemCodeConstraint
+{
+    public string $regexMessage = 'System name can have only letters, digits or underscore symbol';
+
+    public int $min = AttributeCode::MIN_LENGTH;
```

Comment:
Those values are not inherited?

Reply:
`$min` Yes, in this time. 
`$regexMessage` is specyfic for Attribute.